WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -523,6 +523,11 @@ private extension OrderDetailsViewController {
         let reviewOrderViewModel = ReviewOrderViewModel(order: viewModel.order, products: viewModel.products, showAddOns: viewModel.dataSource.showAddOns)
         let controller = ReviewOrderViewController(viewModel: reviewOrderViewModel) { [weak self] in
             guard let self = self else { return }
+            ServiceLocator.analytics.track(.orderStatusChange, withProperties: [
```

Comment:
The indentation here doesn't look too bad, but [putting each argument on a new line](https://github.com/raywenderlich/swift-style-guide/#function-calls) seems to improve it a little

```swift
ServiceLocator.analytics.track(
    .orderStatusChange,
    withProperties: [
        "id": self.viewModel.order.orderID,
        "from": self.viewModel.order.status.rawValue,
        "to": OrderStatusEnum.completed.rawValue
    ])
```

Reply:
Thank you, this really reads much better!